src/components/PopupWithForm.js

```diff
@@ -34,7 +35,14 @@ export default class PopupWithForm extends Popup {
     this.form.addEventListener("submit", (event) => {
       event.preventDefault();
       this._handleFormSubmit(this._getInputValues());
-      this.close();
     });
   }
+
+  renderLoading(isLoading, loadingText = "Saving...") {
+    if (isLoading) {
+      this._submitButton.textContent = loadingText;
+    } else {
+      this._submitButtonText.textContent = this._submitBtnText;
```

Comment:
` this._submitButtonText.textContent `

* There is an error when I submit any form https://snipboard.io/YtP4p6.jpg 


You should use `this._submitButton` here


```js

      this._submitButton.textContent = this._submitBtnText;

```